runtime/queries/nix/injections.scm

```diff
@@ -40,3 +64,89 @@
   (#match? @_path "^text$")
   (#set! injection.language "bash")
   (#set! injection.combined))
+
+; trivial-builders.nix pkgs.writeCBin name content
+((apply_expression
+   function: (apply_expression function: (_) @_func)
+   argument: (indented_string_expression (string_fragment) @injection.content))
+ (#match? @_func "(^|\\.)writeCBin$")
```

Comment:
```suggestion
 (#match? @_func "(^|\\.)writeC(Bin)$")
```

Reply:
Those are equivalent though?

Reply:
As written they should be, perhaps you meant `(Bin)?`?

Reply:
There's no `writeC`, only `writeCBin` in nixpkgs/.../trivial-builders.nix.
It would only makes sense for scripts, not much for compiled C.

Reply:
@pacien you are right, `writeC` has been removed [here](https://github.com/NixOS/nixpkgs/commit/982f46985e37a6488d8e904b46e0cba2060adc71) a while ago. The query works for both. I wouldn't mind keeping it anyway for backwards compat? 🤷🏻  

@nrdxp thanks, thats right.

Reply:
Okay, added!